src/basic/cgroup-util.c

```diff
@@ -2054,7 +2054,9 @@ int cg_unified_cached(bool flush) {
                         unified_systemd_v232 = false;
                 } else {
                         if (statfs("/sys/fs/cgroup/systemd/", &fs) < 0)
-                                return log_debug_errno(errno, "statfs(\"/sys/fs/cgroup/systemd\" failed: %m");
+                                /* Some other software may have set up /sys/fs/cgroup in a configuration we do not recognize. */
+                                return log_debug_errno(SYNTHETIC_ERRNO(ENOMEDIUM),
+                                                       "No known cgroup or cgroup2 hierarchy found");
```

Comment:
hmm, we definitely should log the error we get with %m. hiding errors blindly sucks.

As I understood the original gentoo bug report this is some non-systemd host, i.e. cgroupsv1 without the name=systemd hierarchy?

I figure we can handle that case, but this means we should convert ENOENT to ENOMEDIUM but no other errors.

Reply:
> As I understood the original gentoo bug report this is some non-systemd host, i.e. cgroupsv1 without the name=systemd hierarchy?

Correct.

> I figure we can handle that case, but this means we should convert ENOENT to ENOMEDIUM but no other errors.

Makes sense.